docgen/layouts/example.ejs

```diff
@@ -5,6 +5,8 @@
     <meta charset="utf-8">
     <meta content="width=device-width,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0,user-scalable=no" name="viewport">
     <link rel="icon" href="assets/favicon.png">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/bootstrap/3.3.5/css/bootstrap.min.css" />
```

Comment:
@bobylito You are working on making the templates use pug, It would be also good to be able from an example to inject things in the `<head>` (style, js).